src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/AppointmentsService.kt

```diff
@@ -100,6 +104,26 @@ class AppointmentsService(
     return appointmentList
   }
 
+  fun mapAppointmentsFromDatabase(deliusContacts: List<DeliusContactEntity>) = deliusContacts.map { deliusContact ->
+    val customFieldsFromNotes = getCustomFieldsFromNotes(deliusContact.notes, deliusContact.id)
+    Appointment(
+      title = extractSectionFromNotes(customFieldsFromNotes, APPOINTMENT_TITLE, deliusContact.id), // TODO put all sections in constants
```

Comment:
is this TODO still needed? if so can we raise a tech debt ticket?

Reply:
Fixed